src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -818,9 +837,9 @@ public abstract partial class XmlReader : System.IDisposable
         public abstract bool Read();
         public virtual System.Threading.Tasks.Task<bool> ReadAsync() { throw null; }
         public abstract bool ReadAttributeValue();
-        public virtual object ReadContentAs(System.Type returnType, System.Xml.IXmlNamespaceResolver namespaceResolver) { throw null; }
+        public virtual object ReadContentAs(System.Type returnType, System.Xml.IXmlNamespaceResolver? namespaceResolver) { throw null; }
         [System.Diagnostics.DebuggerStepThroughAttribute]
-        public virtual System.Threading.Tasks.Task<object> ReadContentAsAsync(System.Type returnType, System.Xml.IXmlNamespaceResolver namespaceResolver) { throw null; }
+        public virtual System.Threading.Tasks.Task<object> ReadContentAsAsync(System.Type returnType, System.Xml.IXmlNamespaceResolver? namespaceResolver) { throw null; }
```

Comment:
Most likely couple of namespaceResolvers below should be nullable, i.e.:
`ReadElementContentAs`

Reply:
Seems it will throw InvalidOperationException in some cases when this is null

Reply:
Yes, that's why I didn't change those signatures on https://github.com/dotnet/runtime/pull/41079